pos_payment_method_cashdro/static/src/js/screens.js

```diff
@@ -0,0 +1,243 @@
+odoo.define("pos_payment_method_cashdro.screens", function (require) {
+    "use strict";
+
+    var screens = require("point_of_sale.screens");
+    var core = require("web.core");
+
+
+    screens.PaymentScreenWidget.include({
+
+        render_paymentlines : function(){
+            this._super.apply(this, arguments);
+            var _this = this;
```

Comment:
It's kind of weird to see `_this` as a reference to `this` in js.
The "community standard" would be `self`, which is also more readable due to the lack of underscore

```suggestion
            const self = this;
```

---

Also, not related to only this line, you might want to consider using `const` for pretty much all `var` usages
- [JavaScript ES6+: var, let, or const?](https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75)

Reply:
Well, `self` is indeed an actual const for `window` in [JS](https://developer.mozilla.org/en-US/docs/Web/API/Window/self), so `_this` is a usual convention to keep the pointer of the previous `this` scope.

> Also, not related to only this line, you might want to consider using const for pretty much all var usages

Well, I was developing this with ES5 standard before the recent commitment to drop that constraint... so I will just keep it like that for this version, and will move on to moder JS in v13 :slightly_smiling_face: